sdk/src/azure/core/az_http_policy_logging.c

```diff
@@ -219,9 +219,9 @@ void _az_http_policy_logging_log_http_request(az_http_request const* request)
 }
 
 void _az_http_policy_logging_log_http_response(
-    az_http_response const* response,
+    az_http_response const* const response,
     int64_t duration_msec,
-    az_http_request const* request)
+    az_http_request const* const request)
 {
   uint8_t log_msg_buf[AZ_LOG_MESSAGE_BUFFER_SIZE] = { 0 };
   az_span log_msg = AZ_SPAN_FROM_BUFFER(log_msg_buf);
```

Comment:
I notice core/az_http_policy_retry.c was skipped. Same question as my other comments about making the status_codes pointer const on line 88: 
AZ_INLINE AZ_NODISCARD az_result _az_http_policy_retry_get_retry_after(
    az_http_response* ref_response,
    az_http_status_code const* status_codes,
    bool* should_retry,
    int32_t* retry_after_msec)

Reply:
Same response as above. Status codes here points to an array of elements, just like log classifications. Great questions though, keep them coming :)